module_utils/network_lsr/argument_validator.py

```diff
@@ -465,6 +475,22 @@ def _validate_post(self, value, name, result):
 
 
 class ArgValidator_DictIP(ArgValidatorDict):
+    VALID_MODES = [
+        "debug",
```

Comment:
Please sort this list

Reply:
sorted

Reply:
Till, I suddenly remerbered, the previous order before sorting is the same order as we defined in NM